src/main/scala/geotrellis/sdg/OutputPyramid.scala

```diff
@@ -29,15 +37,20 @@ object OutputPyramid {
     val pyramid = Pyramid.fromLayerRDD(baseLayer, Some(baseZoom), Some(0))
 
     pyramid.levels.foreach { case (zoom, tileRdd) =>
-      val colorRamp = ColorRamps.Viridis
-      //val Some((min, max)) = histogram.minMaxValues()
-      //val breaks = for { break <- (0 to 64)} yield min + (break * (max-min) / 64)
-      val colorMap = colorRamp.toColorMap(histogram)
+      val Some((min, max)) = histogram.minMaxValues()
+      val width = (max - min) / colorRamp.numStops
+      val breaks = (0 to colorRamp.numStops).map { v => v * width }.toArray
```

Comment:
@jfrankl noted that only four colors are ever shown with this setup. Looks like ColorMap ClassBoundaryType defaults to LessThanOrEqualTo. Which means our first break is less than or equal to zero with this code and is consistent with the observation that the lightest color is never visible. Reminder to fix and test this before merging by setting the range to `(1 to colorRamp.numStops + 1)` instead.